packages/components/src/templates/next/components/complex/Hero/HeroGradient.tsx

```diff
@@ -0,0 +1,75 @@
+import type { HeroProps } from "~/interfaces/complex/Hero"
+import { getReferenceLinkHref, isExternalUrl } from "~/utils"
+import { ComponentContent } from "../../internal/customCssClass"
+import { LinkButton } from "../../internal/LinkButton/LinkButton"
+
+export const HeroGradient = ({
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+  site,
+  LinkComponent,
+}: HeroProps) => {
+  const backgroundSrc =
+    isExternalUrl(backgroundUrl) || site.assetsBaseUrl === undefined
+      ? backgroundUrl
+      : `${site.assetsBaseUrl}${backgroundUrl}`
+
+  return (
+    <section
+      className="flex min-h-[15rem] bg-cover bg-center bg-no-repeat sm:min-h-[22.5rem] lg:min-h-[31.25rem]"
+      style={{
+        backgroundImage: `url('${backgroundSrc}')`,
+      }}
+    >
+      <div className="w-full content-center bg-gradient-to-r from-[rgba(0,0,0,85%)] to-[rgba(0,0,0,10%)] xl:from-[rgba(0,0,0,100%)]">
+        <div
+          className={`${ComponentContent} flex flex-row justify-start py-16 text-start text-base-content-inverse`}
+        >
+          <div className="xl:max-w-50% flex w-full flex-col gap-9 sm:w-3/5">
+            <div className="flex flex-col gap-6">
+              <h1 className="prose-display-xl break-words">{title}</h1>
+              {subtitle && <p className="prose-title-lg-regular">{subtitle}</p>}
+            </div>
+            {buttonLabel && buttonUrl && (
```

Comment:
[clarification] not sure if i'm seeing things wrong but this conditional looks like it applies across both the main and secondary button? ie, we'd only show the secondary button fi the main button si shown. this makes sense to me but we need to make sure that wording is clear on studio so that users aren't confused by this behaviour

Reply:
ah yes, i don't quite know the context/rationale behind this logic, but its been there for for Block and Gradient variance already, so I copied it

Reply:
i think it makes sense and it reads as "don't show the secondary button if i don't hvae a primary button" which seems to be reasonable